packages/app/src/app/overmind/internalActions.ts

```diff
@@ -286,13 +279,6 @@ export const setCurrentSandbox: AsyncAction<Sandbox> = async (
   });
 
   effects.executor.setupExecutor();
-
-  /*
-    There seems to be a race condition here? Verify if this still happens with Overmind
-  */
-  if (oldSandboxId !== state.editor.currentId) {
-    delete state.editor.sandboxes[oldSandboxId];
```

Comment:
Did we verify this? I think the case is when you press the back button after navigating from sandbox to sandbox we don't refetch the sandbox info, or we don't join the team live session.

Reply:
Whenever we change Sandbox now it runs the exact same logic. It downloads it, sets it, restarts live etc. There is no difference it being there or not. So yeah, I do not see why we need this logic 😄